app/board.rb

```diff
@@ -0,0 +1,93 @@
+class board
+    attr_accessor :squares
+
+    def initialize(player_1, player_2)
+        @player_1 = player_1
+        @player_2 = player_2
+        @squares = Array.new(64, " ")
+        start_pos
+
+
+        @positions = [  [a,1], [a,2],[a,3],[a,3],[a,4],[a,5],[a,6],[a,7], [a,8], 
```

Comment:
Yeah so this is the most interesting part of this project. Here we're asked to translate a real world thing (A chess board) into something that can be represented with the data structures we have available (lists and hashes).

Naturally, we have quite a few options for how we can do this, so I'd like to start some discussion around this.

Here it looks like you've got two things, `@squares` which is your board, represented as a flat array. Then `@positions` which seems to be a constant enumerating all the possible positions. Could we get away without `@positions`? Maybe if we were able to convert, for example `a` -> `0` we could just use indices to refer to spaces. 

From there, we could keep our `@squares` array a flat, 64 element array. Or we could convert it to a 2D 8x8 array, that way we can refer to pieces directly by index i.e. `@squares[0][0]` would be a rook, and translate to `A1`. We'd some display logic to convert the indices -> Letters and 1-indexed numbers at the very end, but for the rest of the program, we could very well get away with just using indices.